<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="ImgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="chat_lbl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="chat_lbl2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
   <widget class="QLabel" name="chat_lbl3">
     <property name="geometry">
       <rect>
         <x>20</x>
         <y>570</y>
         <width>461</width>
         <height>16</height>
       </rect>
     </property>
     <property name="text">
       <string>TextLabel</string>
     </property>
   </widget>
   <widget class="QLabel" name="end_lbl">
     <property name="geometry">
       <rect>
         <x>400</x>
         <y>510</y>
         <width>461</width>
         <height>16</height>
       </rect>
     </property>
     <property name="text">
       <string>TextLabel</string>
     </property>
   </widget>
  <widget class="QListWidget" name="att_log">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>301</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
